### PR TITLE
Docker updates to node v20 and tilt dev local changes

### PR DESCRIPTION
**Pull request by hselvakumar**
Node V20 Changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,15 +17,15 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.147",
-    "@companieshouse/node-session-handler": "^5.0.1",
-    "@companieshouse/structured-logging-node": "^2.0.1",
+    "@companieshouse/api-sdk-node": "^2.0.103",
+    "@companieshouse/node-session-handler": "^4.1.9",
+    "@companieshouse/structured-logging-node": "^1.0.8",
     "cookie-parser": "1.4.5",
```
Comment (mwejuli-ch) [inline package.json:23]: shouldn't we be using the latest versions of CH packages here?

File: package.json [hunk 2]
```diff
@@ -17,15 +17,15 @@
...
     "escape-html": "1.0.3",
     "express": "~4.18.2",
     "express-actuator": "^1.8.4",
     "express-validator": "^7.0.1",
-    "govuk-frontend": "^4.7.0",
```
Comment (mwejuli-ch) [inline package.json:28]: `4.7.0` should be compatible with v20.. curious to know why we're downgrading here

**Review by mwejuli-ch — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hselvakumar) [reply package.json:23]: ah ok i took this branch before my previous feature branch gone IN which was CC-1149 thats why its reverting all changes

File: package.json (hunk 2 above)
Comment (hselvakumar) [reply package.json:28]: same as above

**Review by mwejuli-ch — APPROVED** [review]

**Review by caismith-ch — APPROVED** [review]

**Pull request merged**